.gitignore

```diff
@@ -17,6 +17,7 @@
 .env.development.local
 .env.test.local
 .env.production.local
+.idea
```

Comment:
This should be a global ignore in your home environment. See [this tutorial](https://aulasoftwarelibre.github.io/taller-de-git/usobasico/#ignorando-archivos-globalmente).